tests/mvp-demo/mvp-demo.go

```diff
@@ -119,26 +119,19 @@ var _ = framework.MvpDemoSuiteDescribe("MVP Demo tests", Label("mvp-demo"), func
 		_, err = f.AsKubeAdmin.ReleaseController.CreateReleasePlanAdmission("demo", userNamespace, appName, managedNamespace, "", "", "mvp-strategy")
 		Expect(err).NotTo(HaveOccurred())
 
-		defaultEcPolicy := ecp.EnterpriseContractPolicySpec{
+		defaultEcPolicy, err := kc.GetEnterpriseContractPolicy("default", "enterprise-contract-service")
+		Expect(err).NotTo(HaveOccurred())
+
+		defaultEcPolicySpec := ecp.EnterpriseContractPolicySpec{
 			Description: "Red Hat's enterprise requirements",
 			PublicKey:   string(publicKey),
-			Sources: []ecp.Source{
-				{
-					Name: "ec-policies",
-					Policy: []string{
-						"git::https://github.com/enterprise-contract/ec-policies.git//policy",
-					},
-					Data: []string{
-						"git::https://github.com/enterprise-contract/ec-policies.git//data",
-					},
-				},
-			},
+			Sources:     defaultEcPolicy.Spec.Sources,
 			Configuration: &ecp.EnterpriseContractPolicyConfiguration{
```

Comment:
ok, but why is that not working on the mvp demo? That seems like something that should be fixed.

There are other things in the default policy that you may care about:

```
  configuration:
    include:
      - "@minimal"
    exclude:
      # This can be removed once https://issues.redhat.com/browse/OCPBUGS-8428 is addressed
      - step_image_registries
```

Any way that we can update this test to take what is in the default ecp, then add the additional `exclude` item, `cve` to it?

Reply:
@lcarva sorry for coming back late. I went through the [mvp-demo chapter 1 description](https://github.com/redhat-appstudio/e2e-tests/tree/main/tests/mvp-demo#description) and find this 

> For simulating the failed release we are using a custom docker-build template that contains a container image coming from ["untrusted" container image registry](https://github.com/enterprise-contract/ec-policies/blob/2d9fc8317a6349a4a9a1969f16c90dfec4448cd3/data/rule_data.yml#L9-L18). This should guarantee that the EC validation will fail and cause the failure of the release process. 

the default poilcy exclude `step_image_registries` which will pass EC validation, while there is also know cve issues, so in the scenario, customize exclude as "cve" and delete step_image_registries item.

Reply:
ack. makes sense! Thanks for the explanation.